rust/cardano-chain-follower/src/utils.rs

```diff
@@ -1,7 +1,5 @@
 //! Simple general purpose utility functions.
 
-use blake2b_simd::{self, Params};
-
 /// Convert T to an i16. (saturate if out of range.)
 #[allow(dead_code)] // Its OK if we don't use this general utility function.
 pub(crate) fn i16_from_saturating<T: TryInto<i16>>(value: T) -> i16 {
```

Comment:
This should be obsolete now.  We have a totally generic anything to anything from_saturiating implementation we should use instead.